[package] qt/5.15.11: missing plugin

### Description

QSvgPlugin and QSvgIconPlugin are not exposed.

### Package and Environment Details

* Package Name/Version: **qt/5.15.11**
* Operating System+version: **Windows 10**
* Compiler+version: **MSVC 193**
* Conan version: **conan 2.0.13**
* Python version: **Python 3.11.4**


### Conan profile

Host profile:
[settings]
arch=x86_64
build_type=Release
compiler=msvc
compiler.cppstd=14
compiler.runtime=dynamic
compiler.runtime_type=Release
compiler.version=193
os=Windows

Build profile:
[settings]
arch=x86_64
build_type=Release
compiler=msvc
compiler.cppstd=14
compiler.runtime=dynamic
compiler.runtime_type=Release
compiler.version=193
os=Windows

### Steps to reproduce

Add this to CMakeLists.txt, configure will fail
```
target_link_libraries(${PROJECT_NAME} Qt5::QSvgIconPlugin Qt5::QSvgPlugin)
```

Add this to code, build will fail (given that above line is not in CMakeLists.txt)
```
Q_IMPORT_PLUGIN (QSvgPlugin);
Q_IMPORT_PLUGIN (QSvgIconPlugin);
```

### Logs

If import plugin is used
```
[build] main.obj : error LNK2019: symbole externe non résolu "struct QStaticPlugin const __cdecl qt_static_plugin_QSvgPlugin(void)" (?qt_static_plugin_QSvgPlugin@@YA?BUQStaticPlugin@@XZ) référencé dans la fonction "void __cdecl `dynamic initializer for 'staticQSvgPluginInstance''(void)" (??__EstaticQSvgPluginInstance@@YAXXZ) [C:\Users\fnadeau\git-github\qt-demo\build\main.vcxproj]
[build] main.obj : error LNK2019: symbole externe non résolu "struct QStaticPlugin const __cdecl qt_static_plugin_QSvgIconPlugin(void)" (?qt_static_plugin_QSvgIconPlugin@@YA?BUQStaticPlugin@@XZ) référencé dans la fonction "void __cdecl `dynamic initializer for 'staticQSvgIconPluginInstance''(void)" (??__EstaticQSvgIconPluginInstance@@YAXXZ) [C:\Users\fnadeau\git-github\qt-demo\build\main.vcxproj]
[build] C:\Users\fnadeau\git-github\qt-demo\build\Release\main.exe : fatal error LNK1120: 2 externes non résolus [C:\Users\fnadeau\git-github\qt-demo\build\main.vcxproj]
```

if target link is used

```
[cmake] CMake Error at CMakeLists.txt:24 (target_link_libraries):
[cmake]   Target "main" links to:
[cmake] 
[cmake]     Qt5::QSvgIconPlugin
[cmake] 
[cmake]   but the target was not found.  Possible reasons include:
[cmake] 
[cmake]     * There is a typo in the target name.
[cmake]     * A find_package call is missing for an IMPORTED target.
[cmake]     * An ALIAS target is missing.
```

--- a/recipes/qt/5.x.x/conanfile.py
+++ b/recipes/qt/5.x.x/conanfile.py
@@ -1253,6 +1253,8 @@ Examples = bin/datadir/examples""")
 
         if self.options.qtsvg and self.options.gui:
             _create_module("Svg", ["Gui"])
+            _create_plugin("QSvgIconPlugin", "qsvgicon", "iconengines", [])
+            _create_plugin("QSvgPlugin", "qsvg", "imageformats", [])
 
         if self.options.qtwayland and self.options.gui:
             _create_module("WaylandClient", ["Gui", "wayland::wayland-client"])

